clients/go/pkg/commands/delete_resource_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright © 2018 Camunda Services GmbH (info@camunda.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"context"
+	"github.com/camunda/zeebe/clients/go/v8/internal/mock_pb"
+	"github.com/camunda/zeebe/clients/go/v8/internal/utils"
+	"github.com/camunda/zeebe/clients/go/v8/pkg/pb"
+	"github.com/golang/mock/gomock"
+	"testing"
+)
+
+func TestDeleteResourceCommand(t *testing.T) {
```

Comment:
🔧 I have seen the other tests and know they look similar, still I would like to have some better separation of what is given, what are we doing and what we verify. I mean the given, when, then blocks.

Reply:
❌ / 🔧  What I miss is for example that we verify that the parameter is set as expected on the send command/request. That we can receive the response, I guess on other clients more interesting since we convert this to different types, and here it is empty?)

Would like to see whether timeout works when no response is sent back etc.

Reply:
I get what you're saying, but I gotta admit that at this point I don't know enough Go to do anything else then shamelessly copying what we do for other commands 😅 So I'll say that if it's enough for the other commands it's enough for this one. Maybe we can revisit the entire Go test suite at some point.